### PR TITLE
Remove sccache

### PR DESCRIPTION
**Pull request by Luni-4**
Remove `sccache` in order to fix `cargo-c` tests.

### CODE REVIEW

**Comment by Luni-4** [comment]
@lu-zero 

It's not an `sccache` problem. Do you have any idea?

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -38,63 +38,26 @@ jobs:
...
-        sccache --start-server
-
     - name: Install cargo-c
       run: |
         cargo install cargo-c
 
     - name: Run cargo-c
       run: |
-        cargo cinstall --manifest-path=av_metrics/Cargo.toml \
+        cargo cinstall --all-features \
```
Comment (lu-zero) [inline .github/workflows/ci-build.yml:59]: I wouldn't use `all-features` though.

Comment (Luni-4) [reply .github/workflows/ci-build.yml:59]: I've used `all-features` to fix this warning:
`warning: package replacement is not used: https://github.com/rust-lang/crates.io-index#av-metrics:0.5.0`

**Comment by Luni-4** [comment]
Let's close this one for now

**Pull request closed without merging**